pandas/tests/arrays/test_datetimelike.py

```diff
@@ -1248,8 +1248,8 @@ def test_invalid_nat_setitem_array(arr, non_casting_nats):
 @pytest.mark.parametrize(
     "arr",
     [
-        pd.date_range("2000", periods=4).array,
-        pd.timedelta_range("2000", periods=4).array,
+        pd.date_range("2000", periods=4).array.copy(),
+        pd.timedelta_range("2000", periods=4).array.copy(),
```

Comment:
._values?

Reply:
Yeah, it seems that my test updates are a bit of a mix of both `.array/values.copy()` or `_values`. Will more consistently use `_values`